### DISCUSSION TITLE
Is it safe to remove a prefix ?

### DISCUSSION THREAD

**Discussion by sebsto** [skymethod/op3#38] [General]
Hello, I'm using OP3 for to podcast shows since almost two months and I'm happy with the data I can retrieve and the visualisation.

Currently OP3 redirects to Podtrac that redirects to my source.
I'm happy to the extent I'm considering to remove the redirect to Podtrac and keep only OP3.

Question : will this change in the URLs impact the data collection made by OP3 ? Will it continue to recognize my shows and continue to aggregate data for them ?  I'm not using the podcast UUID in the params.

Thank you or your work on this project !

**Comment by johnspurlock-skymethod** [comment]
Thanks, glad you are finding OP3 useful, this is exactly why it exists.  Removing the podtrac prefix will not have any effect on OP3's matching, when you update your rss feed enclosure urls, OP3 will match the new urls to the same episode.

Whenever you change enclosure urls (for any reason), some apps (notably Spotify) may redownload episodes from users that had already downloaded the old version.  So you may see a slight increase in Spotify downloads, temporarily, depending on how many listeners to your show use Spotify.

Hope that helps!

**Reply by sebsto** [reply]
Thank you John for your quick reply and for the context.
On a side note, once my podtrac subscription will expire, I'll transfer the budget to OP3 :-)

**Reply by johnspurlock-skymethod** [reply]
I like the sound of that - that's what keeps this project going
